desktop/src/backends/ui.rs

```diff
@@ -249,11 +256,71 @@ impl UiBackend for DesktopUiBackend {
 
     fn load_device_font(
         &self,
-        _name: &str,
-        _is_bold: bool,
-        _is_italic: bool,
-        _register: &mut dyn FnMut(FontDefinition),
+        name: &str,
+        is_bold: bool,
+        is_italic: bool,
+        register: &mut dyn FnMut(FontDefinition),
     ) {
+        let families = &match name {
+            "_serif" | "Times New Roman" | "" => vec![
```

Comment:
_serif/etc should be resolved by ruffle before it reaches the backend - opinions on what they should be should be set through `Player::set_default_font`

Reply:
Good idea.